app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInActivity.kt

```diff
@@ -178,6 +197,28 @@ class LoggedInActivity : BaseActivity(R.layout.activity_logged_in) {
                 }
             }
         }
+
+        if (shouldShowTooltip(getLastOpen())) {
+            if (LoggedInTabs.fromId(binding.bottomNavigation.selectedItemId) == LoggedInTabs.HOME) {
+                Handler(Looper.getMainLooper()).postDelayed({
```

Comment:
What does using `Looper.getMainLooper()` do here?

Reply:
Nothing that was needed 😅